### ISSUE TITLE
Add notification of change in content

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2754]
As per Trusted Tester, The page must provide notification of each form-related change in content. 

[Explore Data page has two violations:](https://revenuedata.doi.gov/explore?dataType=Revenue&location=NF%2CNA&mapLevel=State&offshoreRegions=false&period=Calendar%20Year&year=2022)
- As selections are made to dropdown and year sliders, map changes automatically
- Clicking on the map doesn't tell me what it's going to do

Both of these can be solved by adding a message on what is going to change when selections are made

**Comment by lpgoldstein** [comment]
No longer relevant
